tox.ini

```diff
@@ -10,6 +10,7 @@ envlist =
     py39
     py310
     py311
+    py312
```

Comment:
After this release we should probably drop support for Python 3.8 which is EOL. 3.9 will follow at the end of next year.